### PR TITLE
gh-pages: Update Jekyll

### PR DESCRIPTION
**Pull request by crhallberg**
Jekyll was out of date enough to make standing up a dev instance difficult. These updates help enormously.

### CODE REVIEW

**Comment by crhallberg** [comment]
I see now that 025c5a3, 425e994, fa69623 are accessibility fixes. I can cherry-pick those to another branch, since I have more accessibility changes coming for the wiki.

DONE.

File: .github/workflows/jekyll.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["update-v231"]
```
Comment (demiankatz) [inline .github/workflows/jekyll.yml]: Should this be gh-pages for when we merge this?

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @crhallberg, if you don't mind cherry-picking those accessibility fixes to a separate PR, that seems like a good idea -- then if we have trouble getting these major updates to deploy correctly, we can roll them back without losing the other changes.

See below for a question about the deployment process.

Is there somewhere we can see the updated version of the webpage before we commit to deploying it? Is there a way to confirm that the updated version has successfully deployed after this gets merged?

**Review by demiankatz — APPROVED** [review]
Okay, here goes something....

**Pull request merged**